### ISSUE TITLE
Diff:Switch sides does not announce which side is active to screan JAWS

### ISSUE THREAD

**Issue by zuhairmahd** [microsoft/vscode#252832]
Type: <b>Bug</b>

While comparing two files, it is not possible to know which side you are on when you use the diff:switch sides command.  It would be nice if, at least, the title of the window to reflect which file is the "current" one based on the order its name is announced.

VS Code version: Code - Insiders 1.102.0-insider (b68fa590ef54bc8f07dd79cd3175bf78e0113d4c, 2025-06-27T05:03:40.827Z)
OS version: Windows_NT arm64 10.0.26120
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Snapdragon(R) X 12-core X1E80100 @ 3.40 GHz (12 x 2976)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: unavailable_software|
|Load (avg)|undefined|
|Memory (System)|15.61GB (5.22GB free)|
|Process Argv|. --crash-reporter-id 99fedea4-05e1-45d1-be04-4cb3564d089b|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (37)</summary>

Extension|Author (truncated)|Version
---|---|---
codespaces|Git|1.17.3
copilot|Git|1.338.1648
copilot-chat|Git|0.29.2025062705
remotehub|Git|0.64.0
vscode-pull-request-github|Git|0.113.2025062717
microsoft-testing|ms-|0.1.16
azure-dev|ms-|0.9.0
vscode-azure-github-copilot|ms-|1.0.26
vscode-azureappservice|ms-|0.26.2
vscode-azurecontainerapps|ms-|0.8.3
vscode-azurefunctions|ms-|1.17.3
vscode-azureresourcegroups|ms-|0.10.13
vscode-azurestaticwebapps|ms-|0.13.1
vscode-azurestorage|ms-|0.16.5
vscode-azurevirtualmachines|ms-|0.6.9
vscode-containers|ms-|2.0.3
vscode-cosmosdb|ms-|0.26.0
vscode-dotnet-runtime|ms-|2.3.6
debugpy|ms-|2025.11.2025062501
python|ms-|2025.9.2025062701
vscode-pylance|ms-|2025.6.100
remote-containers|ms-|0.417.0
remote-ssh|ms-|0.121.2025053015
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.99.0
vscode-remote-extensionpack|ms-|0.26.0
azure-repos|ms-|0.40.0
powershell|ms-|2025.3.1
remote-explorer|ms-|0.6.2025051209
remote-repositories|ms-|0.42.0
remote-server|ms-|1.5.2
vscode-node-azure-pack|ms-|1.6.0
vscode-speech|ms-|0.16.0
windows-ai-studio|ms-|0.15.2025062307
pester-test|psp|2023.7.8
vscode-yaml|red|1.18.0
vscode-ai-foundry|Tea|0.6.6

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt551:31249597
vscod805:30301674
c4g48928:30535728
962ge761:30841072
dsvsc014:30777825
dsvsc015:30821418
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:30980852
dwnewjupyter:31046869
pythonrstrctxt:31093868
nativeloc1:31118317
e80f6927:31120813
dwcopilot:31158714
c3hdf307:31184662
6074i472:31201624
dwoutputs:31242946
customenabled:31248079
hdaa2157:31222309
copilot_t_ci:31333650
g012b348:31231168
pythoneinst12:31251391
bgtreat:31252123
c7cif404:31309980
pythonpulldiag:31287486
6gi0g917:31259950
996jf627:31264550
pythonrdcb7:31268811
usemplatestapi:31297334
jdghv92:31317040
747dc170:31275146
pythonfullctx:31296836
6518g693:31302842
generatesymbolt:31280541
convertfstringf:31280702
b99bg931:31306656
usemarketplace:31336439
0g1h6703:31329154
nes-emitfast-1:31333560
55d1c281:31333312
nes-yield-off:31337333
6abeh943:31336334
0c165792:31336556
9c1b2907:31336598
nes-conv-11:31337514

```

</details>

**Comment by zuhairmahd** [comment]
More information:
JAWS 2025 May release running on an Surface Pro 11 running the latest version of VSCode insiders.
